docs/evaluate/temporal-cloud/billing-and-cost.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: billing-and-cost
+title: Billing and Cost
+sidebar_label: Billing and cost
+sidebar_position: 7
+description: As an Account Owner, you can access and manage billing details anytime, with invoices available for download on the Billing page. Your billing cycle starts on the 1st of each month, and manual payments are optional if auto-payment is set up.
+slug: /cloud/billing-and-cost
+toc_max_heading_level: 4
+keywords:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+tags:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+---
+
+As an Account Owner, you can view billing and cost details at any time. 
+
+The typical billing cycle begins on the first of the month (UTC). Your first month is prorated based on when you sign up. 
```

Comment:
"Your first month is prorated based on when you sign up" --> I think we are talking about the minimum support fee, right? We should probably call that out explicitly to avoid confusion

Reply:
Changing to "The minimum support fee for your first month is prorated based on when you sign up." Is this okay with you @mastermanu and @UPatel12

Reply:
that looks good @angelazhou32